client/src/components/Timetable/Viewer/TimetableViewer.js

```diff
@@ -28,6 +28,7 @@ export default function TimetableViewer() {
   const dispatch = useDispatch();
   const subjects = useSelector(state => state.subjects);
   const timetable = useSelector(state => state.timetable);
+  const is24 = useSelector(state => state.timeFormat === '24hr');
```

Comment:
Really small nit pick: `use24HourTime` or something similar is a better variable name (not immediately clear what 'is24' refers to when skimming (though yes, it's eventually quite apparent it's talking about 24 hour time).